internal/saucecloud/imagerunner.go

```diff
@@ -558,7 +656,7 @@ func startProgressTicker(ctx context.Context, progress *int) (cancel context.Can
 			case <-ctx.Done():
 				return
 			case <-t.C:
-				log.Info().Msgf("Suites in progress: %d", *progress)
+				log.Info().Msgf("[notice] Suites in progress: %d", *progress)
```

Comment:
Trying this PR out locally, I found this progress ticker actually rather distracting, interfering. In case of active live logs, perhaps it's better to not long anything via the progress ticker at all 🤔